src/ExtensionsSample/Samples/EasyTableSamples.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+using Microsoft.WindowsAzure.MobileServices;
+using WebJobs.Extensions.EasyTables;
+
+namespace ExtensionsSample
+{
+    public static class EasyTableSamples
+    {
+        public static void InsertItem([EasyTable] out Item newItem)
+        {
+            newItem = new Item()
+            {
+                Id = Guid.NewGuid().ToString(),
+                Text = new Random().Next().ToString()
+            };
+        }
+
+        public static async Task EnqueueItemToProcess(
+            [TimerTrigger("00:01")] TimerInfo timer,
+            [EasyTable] IMobileServiceTableQuery<Item> itemQuery,
```

Comment:
Is the intent for users to be able to use this query model in script (e.g. Node.js)? Wondering how that would work. Maybe not?